Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a provider of reactive extensions.
+///
+/// - note: `ReactiveExtensionProvider` does not indicate whether a type is
+///         reactive. It is intended for extensions to types that do not own by
+///         the module so as to avoid name collision and return type ambiguity.
+public protocol ReactiveExtensionProvider: class {}
```

Comment:
I think we'd rather treat all extension methods and properties as a single bloc under `reactive`.